### PR TITLE
Add podspec :heavy_check_mark:

### PR DESCRIPTION
**Pull request by Pitometsu**
- Fix few unused lint errors.

### CODE REVIEW

**Comment by Pitometsu** [comment]
Please, check and merge to `master` if everything okay.

Also, there's some minor fixes to avoid lint warnings.
So please, tag this commit as `1.0.0` for pods or tag it with something different and notify me to update podspec. Thank you for attention.

**Comment by Pitometsu** [comment]
```sh
 $ pod spec lint

 -> ReactiveObjC (1.0.0)
    - WARN  | xcodebuild:  ReactiveObjC/ReactiveObjC/RACCompoundDisposable.m:85:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveObjC/ReactiveObjC/RACCompoundDisposable.m:132:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveObjC/ReactiveObjC/RACSerialDisposable.m:63:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveObjC/ReactiveObjC/RACSerialDisposable.m:79:12: warning: unused variable 'result' [-Wunused-variable]

Analyzed 1 podspec.

[!] The spec did not pass validation, due to 4 warnings (but you can use `--allow-warnings` to ignore them).
```

File: ReactiveObjC.podspec [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                   DESC
+
+  s.homepage     = "https://reactivecocoa.io"
+  s.screenshots  = "https://reactivecocoa.io/img/logo.png"
+  s.license      = { type: "MIT", file: "LICENSE.md" }
+
+  s.documentation_url  = "https://github.com/ReactiveCocoa/ReactiveObjC/"\
+                         "tree/master/Documentation#readme"
+
+  s.author             = { "Josh Abernathy" => "josh@github.com" }
```
Comment (mdiep) [inline ReactiveObjC.podspec]: Can you set the author to be `ReactiveCocoa` to match [what ReactiveSwift does](https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/ReactiveSwift.podspec).

File: ReactiveObjC.podspec [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                            "UIButton,UICollectionReusableView,UIControl,UIDatePicker,"\
+                            "UIGestureRecognizer,UIImagePicker,UIRefreshControl,"\
+                            "UISegmentedControl,UISlider,UIStepper,UISwitch,UITableViewCell,"\
+                            "UITableViewHeaderFooterView,UIText,MK,AppKit,NSControl,NSText,"\
+                            "NSTable,NSURLConnection}*"
+
+  s.requires_arc = true
+
+  s.frameworks   = "Foundation"
+  s.dependency     "Result", "~> 2.0"
```
Comment (mdiep) [inline ReactiveObjC.podspec]: ReactiveObjC doesn't depend on Result.

**Comment by Pitometsu** [comment]
@mdiep done, please, check https://github.com/ReactiveCocoa/ReactiveObjC/pull/11/commits/b4ad8ce34b6b158db3064de1bf9399d9d54884ec.

File: ReactiveObjC.podspec (hunk 1 above)
Comment (Pitometsu) [inline ReactiveObjC.podspec]: https://github.com/ReactiveCocoa/ReactiveObjC/pull/11/commits/c302fbece3f9a8f667a575189152c72dca74adeb#diff-8d0801a224ad661e150ea9ba9abcde76R21

File: ReactiveObjC.podspec (hunk 2 above)
Comment (Pitometsu) [inline ReactiveObjC.podspec]: https://github.com/ReactiveCocoa/ReactiveObjC/pull/11/commits/c302fbece3f9a8f667a575189152c72dca74adeb#diff-8d0801a224ad661e150ea9ba9abcde76L60

**Comment by mdiep** [comment]
Thanks!

**Comment by ikesyo** [comment]
I've released 1.0.1 with the build warning fixes and push the podspec to CocoaPods trunk. 🚀

**Pull request merged**